api/commands/compile.ts

```diff
@@ -73,3 +84,30 @@ class CompileChildProcess {
     }
   }
 }
+
+function checkDataformJsonValidity(dataformJsonParsed: { [prop: string]: string }): string {
+  const invalidWarehouseProp = function(): string {
```

Comment:
nit: the linter should be complaining here; better style is to use `() => { ... }` than `function(): { ... }` because the fat-arrow operator correctly binds `this`.

Reply:
Done (though linter was not complaining).